documentation.yaml

```diff
@@ -0,0 +1,66 @@
+apiDoc:
+  title: wix-ui-tpa
+  description: Our official library of React components that are designed to be used in Wix TPA's.
+  category: fe-guild
+  status: ga
+  gitUrl: js
+  markdownOnlyAutoUpdate: true
+  docs:
+    - file: README.md
+      title: Getting Started
+    - file: docs/USAGE.md
+      title: Usage
+    - file: Contribution/docs/CONTRIBUTION.md
+      title: Contribution/How To Contribute
+    - file: docs/COMPONENT_GUIDELINES.md
+      title: Contribution/Guidelines
+    - file: docs/MIGRATING-V1-V2.md
+      title: Migration/V1 to V2
+    - file: docs/RTL.md
+      title: RTL Support
+    - file: docs/internal/VERSION_RELEASE.md
+      title: Version Release Instructions
+    - file: CHANGELOG.md
+      title: Change Log/V2
+    - file: CHANGELOG-v1.md
+      title: Change Log/V1
+    - file: src/components/Tabs/README.md
+      title: components/Tabs
+    - file: src/components/Dropdown/README.md
+      title: components/Dropdown
+    - file: src/components/Card/README.md
+      title: components/Card
+    - file: src/components/SectionNotification/README.md
+      title: components/SectionNotification
+    - file: src/components/CheckboxGroup/README.md
+      title: components/CheckboxGroup
+    - file: src/components/Input/README.md
+      title: components/Input
+    - file: src/components/OverlappingCard/README.md
+      title: components/OverlappingCard
+    - file: src/components/StatesButton/README.md
+      title: components/StatesButton
+    - file: src/components/Ratings/README.md
+      title: components/Ratings
+    - file: src/components/Checkbox/README.md
+      title: components/Checkbox
+    - file: src/components/DotNavigation/README.md
+      title: components/DotNavigation
+    - file: src/components/StripCard/README.md
+      title: components/StripCard
+    - file: src/components/Divider/README.md
+      title: components/Divider
+    - file: src/components/RadioButtonGroup/README.md
+      title: components/RadioButtonGroup
+    - file: src/components/IconButton/README.md
+      title: components/IconButton
+    - file: src/components/NewCard/README.md
+      title: components/NewCard
+    - file: src/components/Counter/README.md
+      title: components/Counter
+    - file: src/components/IconToggle/README.md
+      title: components/IconToggle
+    - file: src/components/FloatingDropdown/README.md
+      title: components/FloatingDropdown
+    - file: src/components/Grid/README.md
+      title: components/Grid
```

Comment:
@jonathanadler I'm removing it, but do you have an alternative for those docs?

Funny thing about this list, I've use a shell command to generate it 🙈 

```sh
find src/components -name "*.md" | jq -Rn '[inputs | { file: ., title: . | sub("src/" ; "") | sub("/README.md";"") }]' | yq -y
```

```suggestion
```

Reply:
we use storybook for documentation (all the `*.index.story.tsx` files), which have examples and stuff like that.
I wonder if there's a way to integrate them with the internal docs.

Reply:
No need to publish the components documentation. for this, we have the storybook which provides good documentation for components. WixDocs will not compete with that, so it should get only the standard MD files

Reply:
Thanks for explaining 😄 
So I guess we should take it with WixDocs to have an integration with Storybook 🙏 

cc: @alon-cohen

Reply:
I agree with the guys, we are not trying to solve what the storybook gives when speaking about components.

However, we should think about the flows that engineers might look for that information inside Wix Docs.
Maybe we can have 1 readme giving general info about the use-cases and a link to storybook?

WDYT? @shlomitc @wildcard @jonathanadler @itaiche

Reply:
@alon-cohen This should be reflected in the main `README.md` file

Reply:
Sounds good! So the [README](https://github.com/wix/wix-ui-tpa/blob/master/README.md) should have a clear link to Storybook & explaining that on storybook you will find the entire catalog of the library's components.

> IMHO I still think WixDocs could parse the Storybook & create an index of everything (One place to search all components)